integreat_cms/core/signals/hix_signals.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from ...cms.models import PageTranslation
+from ...cms.views.utils.hix import lookup_hix_score
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=PageTranslation)
+def page_translation_save_handler(instance, **kwargs):
+    r"""
+    Calculates a hix store for a page translation after saving
+
+    :param instance: The page translation that gets saved
+    :type instance: ~integreat_cms.cms.models.pages.page_translation.PageTranslation
+
+    :param \**kwargs: The supplied keyword arguments
+    :type \**kwargs: dict
+    """
+    if instance.hix_score or not hix_enabled(instance):
```

Comment:
```suggestion
    if kwargs.get("raw") or instance.hix_score or not hix_enabled(instance):
```
We have to ignore raw save signals (e.g. during fixture loading), because they will cause errors when loading the test data into a fresh database.

> `raw`
> 
> A boolean; `True` if the model is saved exactly as presented (i.e. when loading a fixture). One should not query/modify other records in the database as the database might not be in a consistent state yet.

– https://docs.djangoproject.com/en/3.2/ref/signals/#post-save